plugins/experimental/ssl_session_reuse/src/publish.cc

```diff
@@ -117,8 +117,8 @@ RedisPublisher::is_good()
 ::redisContext *
 RedisPublisher::setup_connection(const RedisEndpoint &re)
 {
-  ::pthread_t my_id = ::pthread_self();
-  TSDebug(PLUGIN, "RedisPublisher::setup_connection: Called by threadId: %lx", my_id);
+  auto my_id = (unsigned long long)::pthread_self();
+  TSDebug(PLUGIN, "RedisPublisher::setup_connection: Called by threadId: %llx", reinterpret_cast<unsigned long long>(my_id));
```

Comment:
Do we need this cast? Others lines don't have it.

Reply:
Yeah, my bad.  Fixed it.